src/styles/variables/colors.scss

```
@@ -1,71 +1,14 @@
-$default-color-system-red-dark: rgb(255, 69, 58);
-$default-color-system-orange-dark: rgb(255, 159, 10);
-$default-color-system-yellow-dark: rgb(255, 214, 10);
-$default-color-system-green-dark: rgb(48, 209, 88);
-$default-color-system-teal-dark: rgb(100, 210, 255);
-$default-color-system-blue-dark: rgb(10, 132, 255);
-$default-color-system-indigo-dark: rgb(94, 92, 230);
-$default-color-system-purple-dark: rgb(191, 90, 242);
-$default-color-system-pink-dark: rgb(255, 45, 85);
-
-$default-color-system-gray-01-dark: rgb(142, 142, 147);
-$default-color-system-gray-02-dark: rgb(99, 99, 102);
-$default-color-system-gray-03-dark: rgb(72, 72, 74);
-$default-color-system-gray-04-dark: rgb(58, 58, 60);
-$default-color-system-gray-05-dark: rgb(44, 44, 46);
-$default-color-system-gray-06-dark: rgb(28, 28, 30);
-
-// Accessible Colors (Light, Default)
-$accessible-color-system-red: rgb(215, 0, 21);
-$accessible-color-system-orange: rgb(201, 52, 0);
-$accessible-color-system-yellow: rgb(178, 80, 0);
-$accessible-color-system-green: rgb(36, 138, 61);
-$accessible-color-system-teal: rgb(0, 113, 164);
-$accessible-color-system-blue: rgb(0, 64, 221);
-$accessible-color-system-indigo: rgb(54, 52, 163);
-$accessible-color-system-purple: rgb(137, 68, 171);
-$accessible-color-system-pink: rgb(211, 15, 69);
-
-$accessible-color-system-gray-01: rgb(108, 108, 112);
-$accessible-color-system-gray-02: rgb(142, 142, 147);
-$accessible-color-system-gray-03: rgb(174, 174, 178);
-$accessible-color-system-gray-04: rgb(188, 188, 192);
-$accessible-color-system-gray-05: rgb(216, 216, 220);
-$accessible-color-system-gray-06: rgb(235, 235, 240);
-
-// Accessible Colors (Dark)
-$accessible-color-system-red-dark: rgb(255, 105, 97);
-$accessible-color-system-orange-dark: rgb(255, 179, 64);
-$accessible-color-system-yellow-dark: rgb(255, 212, 38);
-$accessible-color-system-green-dark: rgb(48, 219, 91);
-$accessible-color-system-teal-dark: rgb(112, 215, 255);
-$accessible-color-system-blue-dark: rgb(64, 156, 255);
-$accessible-color-system-indigo-dark: rgb(125, 122, 255);
-$accessible-color-system-purple-dark: rgb(218, 143, 255);
-$accessible-color-system-pink-dark: rgb(255, 100, 130);
-
-$accessible-color-system-gray-01-dark: rgb(174, 174, 178);
-$accessible-color-system-gray-02-dark: rgb(124, 124, 128);
-$accessible-color-system-gray-03-dark: rgb(84, 84, 86);
-$accessible-color-system-gray-04-dark: rgb(68, 68, 70);
-$accessible-color-system-gray-05-dark: rgb(54, 54, 56);
-$accessible-color-system-gray-06-dark: rgb(36, 36, 38);
\ No newline at end of file
+$koreasy-color-primary: #44da92
+$koreasy-color-secondary: #64F0AD
+$koreasy-color-black: #1c1c1e
+$koreasy-color-white: #ffffff
+$koreasy-color-gray-00: #535356
+$koreasy-color-gray-01: #8e8e93
+$koreasy-color-gray-02: #aeaeb2
+$koreasy-color-gray-03: #c7c7cc
+$koreasy-color-gray-04: #d1d1d6
+$koreasy-color-gray-05: #e5e5ea
+$koreasy-color-gray-06: #f2f2f7
+$koreasy-color-red: #ff3b30
+$koreasy-color-yellow: #FFC940
```

모든 scss 소스코드에 괄호가 누락되었습니다. 괄호를 추가해주시면 좋겠습니다.
```suggestion
$koreasy-color-primary: #44da92;
$koreasy-color-secondary: #64F0AD;
$koreasy-color-black: #1c1c1e;
$koreasy-color-white: #ffffff;
$koreasy-color-gray-00: #535356;
$koreasy-color-gray-01: #8e8e93;
$koreasy-color-gray-02: #aeaeb2;
$koreasy-color-gray-03: #c7c7cc;
$koreasy-color-gray-04: #d1d1d6;
$koreasy-color-gray-05: #e5e5ea;
$koreasy-color-gray-06: #f2f2f7;
$koreasy-color-red: #ff3b30;
$koreasy-color-yellow: #FFC940;
```